docs/StandardLibraryProgrammersManual.md

```diff
@@ -12,7 +12,7 @@ TODO: Should this subsume or link to [AccessControlInStdlib.rst](https://github.
 1. Library Organization
     1. What files are where
         1. Brief about CMakeLists
-	1. Brief about GroupInfo.json
+    1. Brief about GroupInfo.json
```

Comment:
Did you meant to reduce the indentation here? I think the original indentation is off because it's _missing_ a space, not because it has three too many.

Reply:
The original indentation is a tab, but yeah, that could be intended to be 4 or 8 spaces. I'm inclined to think Xiaodi's right that this was probably intended to be nested, though. Whoops!

Reply:
sorry about the change but aren't tabs evil?  What should I do?

Reply:
I think matching the 8 spaces on the line above is probably the right fix!

Reply:
OK